ui/helpers/constants/common.js

```diff
@@ -13,3 +13,15 @@ export const GAS_ESTIMATE_TYPES = {
   FAST: 'FAST',
   FASTEST: 'FASTEST',
 };
+
+let _supportLink = 'https://support.metamask.io';
+let _supportRequestLink = 'https://metamask.zendesk.com/hc/en-us/requests/new';
+
+///: BEGIN:ONLY_INCLUDE_IN(flask)
+_supportLink = 'https://metamask-flask.zendesk.com/hc';
+_supportRequestLink =
+  'https://metamask-flask.zendesk.com/hc/en-us/requests/new';
```

Comment:
This link does not appear to work :thinking:

Reply:
It redirects to some ZenDesk dashboard, if I'm logged in. If not, it asks me to log in. The page is just missing.

Maybe this needs to get enabled on the Flask ZenDesk instance.

Reply:
Ah yes, I should've noted this in the description. These links exist and will work, but the instance is not open to the public yet!

Reply:
Oh, but I was logged in when I tried and there is still no form. Yet I am able to see https://metamask-flask.zendesk.com/hc

It seemed unlikely that I had permission to view one but not the other.

Reply:
Maybe the form is not accessible until it's open to the public? :shrug: We ought to be able to see it and try it out before it's public though.

Reply:
I'm 99% certain that the URL is correct and that we will be able to try it out before it goes live. I will confirm with Support that this is the case as well.